src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MAGNIFICATION_FACTOR = 2;
+
+    private int size;
+    private int threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
 
     @Override
     public void put(K key, V value) {
-
+        int index = hash(key);
+        if (size == threshold) {
+            toGrow();
+        }
+        Node<K, V> node = table[index];
```

Comment:
you can do like in your method 
public V getValue(K key) {
        Node<K, V> node = table[hash(key)]; 
        
        because int index in you code can be less variable

Reply:
i use Index in public V getValue(K key); just once but in public void put(K key, V value) i use it 2 times, it look more readable